docs/source/tutorial.rst

```diff
@@ -554,7 +488,108 @@ normal. The paragraphs above form a nice guide and checklist for you to follow.
+
+The Action Clients are housed in AgentActionClient nodes, which contains some 
+other useful subscriptions to get information about the field and referee.
+
+At any given time, a given AgentActionClient is playing a single position. 
+Take a look through ``agent_action_client.cpp`` to get an understanding of how
+that works. It creates a Position instance and asks said Position for its task,
+which it then relays to the planner using ROS actions. 
+
+As you can see, strategy decisions are delegated to the Positions. This makes some
+sense with respect to soccer—players play differently based on their position.
+
+There are three major positions: Offense, Defense, and Goalie. You may see
+some others, but these are only for special game cases.
+
+Take some time to read through Offense, Defense, and Goalie, paying special 
+attention to how they each implement ``state_to_task`` and ``update_state``.
+This is called a finite state machine, and it is a crucial concept to get the 
+hang of. Here's `a simple article to get you started.<https://medium.com/@mlbors/what-is-a-finite-state-machine-6d8dec727e2c>`_
+
+Instructions
+~~~~~~~~~~~~
+
+This is the most open-ended part of the tutorial, but you got this! 
+Remember, if you get stuck, ask your peers first. We're a very collaborative
+team. If you're still stuck, your software lead is happy to give you some hints
+and troubleshoot bugs.
+
+Your task is to create a new position, like Offense, Defense, or Goalie. Your
+new position will be called Runner. Note that this class is not a ROS node
+like the last class you made, but it will be a subclass of ``position.hpp``. 
+
+Some useful resources will be the `classes<https://www.learncpp.com/cpp-tutorial/classes-and-class-members/>`_
+and `inheritance<https://www.learncpp.com/cpp-tutorial/basic-inheritance-in-c/>`_ C++ tutorials.
+
+Your runner will be a robot that takes laps around the field. It should run in a rectangle that you choose.
+If you're feeling creative, the shape it runs in can be any polygon with 4 or more sides. 
+
+A runner's process looks like this:
+
+#. Run along first side of shape 
+#. Continue until done
+#. Run along second side of shape
+#. Continue until done
+#. Run along third side of shape
+#. Continue until done
+
+etc, starting over when it finishes the shape.
+
+Hopefully, you're seeing how this list lends nicely to a state machine, where states are sides
+and you know to switch states based on when the robot has reached a vertex (the end of its path).
+
+You will need to look through the other positions to figure out the details of creating this position,
+but here are some more hints.
+
+* The motion command for driving in a straight line is ``"path_target"``.
+* You will probably need to override some methods relating to passing, but you can leave their implementations empty. They don't need to do anything in your position, as your robot will not pass the ball
```

Comment:
I don't understand what you were referring to with this comment.

Reply:
in order to create a new position, virtual methods `derived_acknowledge_pass`, `derived_pass_ball`, and `derived_acknowledge_ball_in_transit` need to be overridden (but their implementations may be left blank for this exercise)